docs/src/description/declarations.md

```diff
@@ -0,0 +1,74 @@
+# Type declaration sections
+
+## trace_columns
```

Comment:
nit: I wonder if we should use just regular text for section names here and later. For example, "Trace columns" instead of "trace_columns".

Reply:
Sure, I'll update